.travis.yml

```diff
@@ -2,7 +2,7 @@
 language: generic
 
 os: osx
-osx_image: beta-xcode6.1
+osx_image: xcode7.3
```

Comment:
Here is the bug report. Looks like there might be a usable patch for this problem.

https://llvm.org/bugs/show_bug.cgi?id=24249